index.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  extend: 'apostrophe-widgets',
+  label: 'Address',
+  addFields: [
+    {
+      name: 'lineOne',
+      label: 'Address Line 1',
+      type: 'string'
+    },
+    {
+      name: 'lineTwo',
+      label: 'Address Line 2',
+      type: 'string'
+    },
+    {
+      name: 'city',
+      label: 'City/Town',
+      type: 'string',
+      required: true
+    },
+    {
+      name: 'state',
+      label: 'State',
+      type: 'string',
+      required: true
+    },
+    {
+      name: 'zip',
+      label: 'Zip/Postal Code',
+      type: 'string',
+      required: true
+    },
+    {
+      name: 'country',
+      label: 'Country',
+      type: 'string'
```

Comment:
how about a `defaultCountry` option at the module level, so that beforeConstruct can find this field with `_.find()` and poke the `def` property onto it appropriately? Then this field can be `required`.

Reply:
Upon reflection, this is a widget, not frontend content entry by the public, and it makes sense to just not list a country when it's obvious from context, so never mind.

Reply:
Ok, thank you for thinking through this.